docs/FAQ.md

```diff
@@ -19,9 +19,9 @@ process, we try to stay on top of these changes and release updates as quickly a
 
 The RAPIDS Accelerator for Apache Spark officially supports:
 - [Apache Spark](get-started/getting-started-on-prem.md)
-- [AWS EMR 6.2.0](get-started/getting-started-aws-emr.md)
-- [Databricks Runtime 7.3](get-started/getting-started-databricks.md)
-- [Google Cloud Dataproc](get-started/getting-started-gcp.md)
+- [AWS EMR 6.2.0, 6.3.0](get-started/getting-started-aws-emr.md)
+- [Databricks Runtime 7.3, 8.2](get-started/getting-started-databricks.md)
```

Comment:
The databricks getting started guide doesn't mention 8.2. Should it?

Reply:
Updates for Databricks in PR #2631